extensions/amp-social-share/amp-social-share.md

```diff
@@ -0,0 +1,80 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-social-share"></a> `amp-social-share`
+
+Displays a social share button.
+
+Example:
+```html
+<amp-social-share type="twitter" width="60" height="44">
+  <script type="application/json">
+    {
+      "text": "The AMP Project",
+      "url": "https://www.ampproject.org/",
+      "attribution": "amphtml"
+    }
+  </script>
+</amp-social-share>
+```
+
+Example with customizations:
+```html
+<amp-social-share type="linkedin" width="60" height="44">
+  <script type="application/json">
+    {
+      "text": "The AMP Project",
+      "url": "https://www.ampproject.org/",
+      "attribution": "amphtml"
+    }
+  </script>
+  <div class="my-style">
+    <a id="customized-social-share" class="custom"><img src="http://example.com/image.jpg"/></a>
+  </div>
+</amp-social-share>
+```
+
+### Structure
+
+Required attributes are `type`, `width` and `height`. The inner `script` tag is also required and it must contain the JSON configuration for the share type ([more on that in the type section](#user-content-types)).
+
+Optionally, you can specify an `anchor` tag without a href set and the social share widget will populate the href for the correct location. This enables customization of the social share element.
+
+### Types
+
+The builtin supported types are configured in [AMP Social Share Config](v.0/amp-amp-social-share-config.js). Below are the possible types and their configuration options:
+- twitter
+  - url `required`
+  - text
+  - attribution
+- facebook
+  - attribution (Your `app_id`) `required`
+  - url
+- pinterest
```

Comment:
So, this is a bit confusing to me. We have a first-class `pinterest` button extension. Does it not cover this?

/cc @cramforce